source/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ endif()
 
 option(USE_SDL2 "Build using SDL2" OFF)
 option(GAME_MODULES_ONLY "Only build game modules" OFF)
+option(BUILD_STEAMLIB "Include the steamlib module (requires Steamworks SDK)" OFF)
```

Comment:
@Warfork this option should be set on builders when you want to include the steamlib module.

Reply:
> @Warfork this option should be set on builders when you want to include the steamlib module.

Noted!